JS/Lesson15/sinan/script.js

```diff
@@ -116,14 +129,49 @@ function checkIfStringIsUpperCase(stringToCheck) {
 	}
 }
 
-checkIfStringIsUpperCase('Write a program that takes in');
-checkIfStringIsUpperCase('write a program that takes in');
+checkIfStringIsUpperCase('Exercise 4.1:','Write a program that takes in');
```

Comment:
Since here you call the function directly and your function takes only one parameter (string value), you should not be adding 'Exercise 4.1:' to it. Instead, if you want it to be printed to the console, you can add it to the test string:
```suggestion
checkIfStringIsUpperCase('Exercise 4.1: Write a program that takes in');
```